tests/cli/device/test_trezor_backend.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from trezorlib import client as trezor_client
+from trezorlib.transport import TransportException
+from usb1 import USBErrorNoDevice, USBErrorBusy
+
+from nucypher.cli.hardware.backends import Trezor
+from nucypher.crypto.signing import InvalidSignature
+
+
+def test_trezor_defaults(mock_trezorlib):
+    trezor_backend = Trezor()
+
+    assert trezor_backend.DEFAULT_BIP44_PATH == "m/44'/60'/0'/0"
+    assert trezor_backend._Trezor__bip44_path == [2147483692, 2147483708,
+                                                  2147483648, 0]
```

Comment:
What do these numbers mean?

Reply:
It's the derived BIP44 path in numerical form.